build-logic/integration-testing/src/main/kotlin/gradlebuild/integrationtests/action/AnnotationGeneratorWorkAction.kt

```diff
@@ -61,8 +79,8 @@ abstract class AnnotationGeneratorWorkAction : WorkAction<AnnotationGeneratorWor
     }
 
     private
-    fun generateGroovyAnnotation(packageName: String): String {
-        @Suppress("UnnecessaryVariable")
+    fun generateGroovyAnnotation(packageName: String, name: String, scriptReceiverAccessors: () -> String): String {
+        @Suppress("GrPackage")
```

Comment:
🤔 Could you please describe which one is this warning?

Reply:
It's this warning:
![image](https://github.com/gradle/gradle/assets/2759152/81e11220-2d0b-49f6-a081-28b2ea270485)

Reply:
Ah, ok! Thanks for showing, as I didn't find too much reference to it.